server/server.js

```diff
@@ -1,8 +1,53 @@
 const express = require('express');
 const port = 3000;
+const database = require('../database/database.js')
 
 const app = express();
 
-app.use(express.static(__dirname + '/../client/dist'));
+app.use(express.static(__dirname + '../client/dist'));
```

Comment:
Consider using the morgan middleware npm module to help with testing if you find it helpful.